03/app.js

```diff
@@ -0,0 +1,60 @@
+const input = document.querySelector('input[type="file"]')
+const imagesList = document.querySelector('.images-list')
+
+input.addEventListener('change', showImage)
+
+function showImage(e) {
+	const files = e.target.files
+
+	Array.from(files).forEach(item => {
+		if (item && item['type'].split('/')[0] === 'image') {
```

Comment:
Bezpieczniej będzie wykorzystać `includes()`: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes

Po prostu sprawdzić czy zawiera słowo `image`. Jeśli chcemy być bardzo dokładni to po prostu zrobiłbym tablicę elementów i sprawdzał czy `item.type` tam występuje :)